cdap-common/pom.xml

```diff
@@ -183,6 +183,12 @@
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
+      <scope>test</scope>
```

Comment:
This is not needed. The scope is defined in the parent `pom`.

Reply:
Thanks, removed.